src/main/java/org/reactome/addlinks/kegg/KEGGReferenceDatabaseGenerator.java

```diff
@@ -245,7 +245,7 @@ public static synchronized String createReferenceDatabaseFromKEGGData(String keg
 		String speciesURL = KEGG_URL.replace("###SP3###", keggPrefix + ":");
 		try
 		{
-			Long dbId = new Long(createReferenceDatabase(newDBName, keggSpeciesName, speciesURL, objectCache));
+			Long dbId = createReferenceDatabase(newDBName, keggSpeciesName, speciesURL, objectCache);
```

Comment:
Not a huge deal, but generally using primitives (i.e. `long` vs `Long`) avoid pitfalls of comparing using `==` if that is ever added.

Reply:
Yes... the Reactome API uses `Long`, though, so that is probably why it was done. I'll have a look.

Reply:
...it gets passed to `adaptor.fetchInstance`, I guess I wanted to match the exact type that the method required.

Reply:
Okay, it's not critical.  You can use `long` if you want.  Java will automatically unbox it.

Reply:
If we ever get to migrate to Java 16+, there are changes coming for primitive wrappers: https://openjdk.java.net/jeps/390

Reply:
Okay, looks like additional reasons to favour primitives instead of classes when possible.

Reply:
Yes. If only we could rewrite the underlying Reactome API...

Reply:
We might be able to make some rewrites for the project in the future.